### PR TITLE
Fix #688: Update extra spacing in topic_lessons_story_summary

### LINKED ISSUE #688: Lowfi: Update topic_lessons_story_summary
Mock: https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/c718fc6d-b336-475a-b31b-fd3bc84882a8/PM-EN-Play-Tab-Progress-1-

In new mock the `topic_lessons_story_summary` has been updated to remove the extra space before chapter names.

Current Develop implementation:
Screenshot 2020-02-22 at 5 49 27 PM

New updated implementation:
Screenshot 2020-02-22 at 5 49 33 PM

So if you see, the space before the chapter names is different in both the mocks.

### PR DESCRIPTION
**Pull request by gunishjain**
## Explanation

 Decreased marginStart to 5 dp, which removed the extra space in left of topic lessons. Could have deleted the marginStart parameter but to be on a safer side provided with little margin.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by gunishjain** [comment]
@rt4914 PTAL. 
I made a fresh PR, sorry for too much delay, this is my first time contributing in such type of big project. Had some trouble at the start but now I think this could solve the issue.

**Review by SayantanBanerjee16 — CHANGES_REQUESTED** [review]
Don't push the auto-generated files generated by Android Studio. Before making a commit, exclude these files and add only those files in which you had done code changes.

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (rt4914) [inline .idea/codeStyles/Project.xml:165]: We don't want any change in `.idea` folder, please remove those files from your commit. (This does not mean that you delete them, it simply means that those files will not be committed when making a PR.)

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="JDK" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (rt4914) [inline .idea/misc.xml:9]: Ditto as abaove.

File: app/src/main/java/org/oppia/app/topic/lessons/ChapterSummaryAdapter.kt [hunk 3]
```diff
@@ -43,4 +43,6 @@ class ChapterSummaryAdapter(
       }
     }
   }
+
+
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/topic/lessons/ChapterSummaryAdapter.kt:47]: Remove these two unnecessary blank lines.

File: app/src/main/res/layout/topic_lessons_story_summary.xml [hunk 4]
```diff
@@ -156,7 +156,7 @@
           android:id="@+id/chapter_recycler_view"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginStart="56dp"
+          android:layout_marginStart="5dp"
```
Comment (rt4914) [inline app/src/main/res/layout/topic_lessons_story_summary.xml]: We keep margin and padding normally in multiple of 4. Therefore 5 is incorrect. At the same time, you can remove this marginStart completely and that would look decent enough for now.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Suggested changes, PTAL (Please take a look)

File: app/src/main/res/layout/topic_lessons_story_summary.xml (hunk 4 above)
Comment (gunishjain) [reply app/src/main/res/layout/topic_lessons_story_summary.xml]: @rt4914 I updated the file with the latest commit, I made a new PR in which I did not include those .idea files , please check if it is is reflecting at your end.

Comment (gunishjain) [reply app/src/main/res/layout/topic_lessons_story_summary.xml]: @rt4914 Made another commit, this time did not include those .idea files and removed marginStart completely.

File: app/src/main/res/layout/topic_lessons_story_summary.xml [hunk 5]
```diff
@@ -156,7 +156,6 @@
           android:id="@+id/chapter_recycler_view"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginStart="5dp"
```
Comment (gunishjain) [inline app/src/main/res/layout/topic_lessons_story_summary.xml]: Please note that this line is removed , but github is showing it exists. Please view the whole file and you will see it is fixed.

File: .idea/misc.xml (hunk 2 above)
Comment (rt4914) [reply .idea/misc.xml:9]: @gunishjain We don't need these changes in these files.

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (rt4914) [reply .idea/codeStyles/Project.xml:165]: @gunishjain We don't need these changes in these files.

**Review by rt4914 — CHANGES_REQUESTED** [review]
My earlier comments have not been solved related to .idea files. Please tak a look.

**Comment by gunishjain** [comment]
@rt4914 closing this PR i guess I messed up a lot. Will try one last time

**Comment by rt4914** [comment]
> @rt4914 closing this PR i guess I messed up a lot. Will try one last time

No worries, you still face issues in this, then ping me on gitter and we can discuss in detail.

**Pull request closed without merging**